# simple message to control wheels. assumes the translation between rads/sec and pwm has already taken place

int32 left_pwm
int32 right_pwm
